index.js

```diff
@@ -9,8 +9,9 @@ function wrapInMediaQuery(node) {
     );
 }
 
-module.exports = postcss.plugin('postcss-require-hover', function () {
-    return function (root) {
+const plugin = () => ({
+    postcssPlugin: 'postcss-require-hover',
```

Comment:
See https://evilmartians.com/chronicles/postcss-8-plugin-migration#step-2-use-the-updated-api